build.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('https');
+const archiver = require('archiver');
+
+https.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions', res => {
+  let data = '';
+    
+  res.on('data', chunk => data += chunk);
+  res.on('end', () => {
+    let content = `const SUPPORTED_EXTENSIONS = ${data};`;
+
+    fs.writeFile("supported_extensions.js", content, e => {
+      if (e) {
+        console.error(e.message);
+      }
+    });
```

Comment:
There could be a race condition here, the rest of the code should go in the callback of `writeFile`, otherwise it could happen that we run `fs.readdir` before the `supported_extensions.js` file is created.